package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "excel",
-  "version": "0.1.7",
+  "version": "0.2.0",
```

Comment:
Since it looks like the public API changed in a backwards-incompatible way (promises instead of callbacks), can you increment the major version (1.0.0)?

Reply:
@trevordixon `0.1.7` -> `0.2.0` already is a major version update.
Do you still want version to be `1.0.0`?

Reply:
Semver < 1.0.0 is a little confusing. I read somewhere that version 1.0.0 defines the public API. The API hasn't changed in some time, so it's probably OK to commit to it now! Yeah, go ahead and make it 1.0.0, and I'll merge it.

Reply:
@trevordixon Ok, done.